fbench/src/fbench/client.cpp

```diff
@@ -220,8 +224,15 @@ Client::run()
                 strcat(_linebuf, _args->_queryStringToAppend.c_str());
             }
             int cLen = _args->_usePostMode ? urlSource.nextContent() : 0;
+            auto content = urlSource.content();
```

Comment:
Specify type explicit as it is not obvious what it is. You are also reassigning it further down.